src/pal/src/CMakeLists.txt

```diff
@@ -198,7 +198,7 @@ set(SOURCES
   thread/tls.cpp
 )
 
-add_library(coreclrpal
+_add_library(coreclrpal
```

Comment:
This adds a static library, I don't think you want to add version to that, since this library gets linked into most of the shared libs. You would end up with the version stuff being twice in them.